src/lib/flows/common/build-chain-flow-helper.js

```diff
@@ -205,9 +206,19 @@ async function checkoutProjectPullRequestFlow(
  * @param {Object} dir the dir to check out
  */
 async function checkoutNode(context, node, checkoutInfo, dir) {
+  const sourceHash = await getRemoteSha(
+    `${context.config.github.serverUrl}/${checkoutInfo.group}/${checkoutInfo.project}`,
+    checkoutInfo.branch
+  );
+
   if (checkoutInfo.merge) {
+    const targetHash = await getRemoteSha(
+      `${context.config.github.serverUrl}/${node.project}`,
+      checkoutInfo.targetBranch
+    );
+
     logger.info(
-      `[${node.project}] Merging ${context.config.github.serverUrl}/${node.project}:${checkoutInfo.targetBranch} into ${context.config.github.serverUrl}/${checkoutInfo.group}/${checkoutInfo.project}:${checkoutInfo.branch}`
+      `[${node.project}] Checking out ${context.config.github.serverUrl}/${node.project}:${checkoutInfo.targetBranch} ${targetHash} and merging ${context.config.github.serverUrl}/${checkoutInfo.group}/${checkoutInfo.project}:${checkoutInfo.branch} ${sourceHash} into '${dir}'`
```

Comment:
@danielezonca if you don't want to check the whole thing just check the log information is the one expected. Thanks